evaluator/src/main/kotlin/DependencyRule.kt

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.evaluator
+
+import com.here.ort.model.Identifier
+import com.here.ort.model.LicenseFinding
+import com.here.ort.model.Package
+import com.here.ort.model.PackageLinkage
+import com.here.ort.model.PackageReference
+import com.here.ort.model.Project
+import com.here.ort.model.Scope
+
+/**
+ * A [Rule] to check a single [dependency][PackageReference].
+ */
+class DependencyRule(
+    ruleSet: RuleSet,
+    name: String,
+    pkg: Package,
+    detectedLicenses: List<LicenseFinding>,
```

Comment:
Dokka would just ignore comments on these parameters, but it lists the related properties from the parent classes as "inherited properties" including documentation. To document the constructor arguments a separate comment for the constructor itself would be required, but this would be mainly duplication.